src/BrazeKit-dev.js

```diff
@@ -635,9 +637,11 @@ function getId() {
 }
 
 function register(config) {
+    var forwarderNameWithVersion = [name, suffix].join('-');
```

Comment:
Would it be more appropriate to call this `forwarderNameWithSuffix`?

```suggestion
    var forwarderNameWithSuffix = [name, suffix].join('-');
```